src/test/kotlin/org/jetBrains/research/tasktracker/tracking/DefaultContentProviderTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.jetBrains.research.tasktracker.tracking
```

Comment:
Why do we actually have jetBrains with a big B? Could you please change it in the src folder and in the test folder